recipe-generator/src/main/resources/templates/ts/ingredient-types.liquid

```diff
@@ -0,0 +1,9 @@
+{%- if ingredient.keyed -%}
+    {%- assign superclass = 'KeyedIngredient' -%}
+{%- else -%}
+    {%- assign superclass = 'Ingredient' -%}
+{%- endif -%}
+import { {{ superclass}} } from "recipe-ts-runtime";
```

Comment:
still using es6 import+export here?

Reply:
Yup the definition files can use those.

Reply:
Or wait, should definitions contain imports?

Reply:
I think so, otherwise you can't properly define types for things in the definition...